torch_geometric/datasets/dgraph.py

```diff
@@ -0,0 +1,97 @@
+import os
+from typing import Callable, Optional
+
+import numpy as np
+import torch
+
+from torch_geometric.data import Data, InMemoryDataset, extract_zip
+from torch_geometric.utils import index_to_mask
+
+
+class DGraphFin(InMemoryDataset):
+    r"""The DGraphFin networks from the
+    `"DGraph: A Large-Scale Financial Dataset for Graph Anomaly Detection"
+    <https://arxiv.org/abs/2207.03579>`_ paper.
+    It is a directed, unweighted dynamic graph consisting of millions of
+    nodes and edges, representing a realistic user-to-user social network
+    in financial industry.
+    Node represents a Finvolution user, and an edge from one
+    user to another means that the user regards the other user
+    as the emergency contact person.
```

Comment:
mention here that the edges have a time attribute, and the granularity of that attribute.